[3.3.0RC1] NPE at OCFileListFragment.exitSelectionMode

via gplay console:
```
java.lang.NullPointerException: 
  at com.owncloud.android.ui.fragment.OCFileListFragment.exitSelectionMode (OCFileListFragment.java:1590)
  at com.owncloud.android.ui.fragment.OCFileListFragment.onFileActionChosen (OCFileListFragment.java:950)
  at com.owncloud.android.ui.fragment.OCFileListFragment.lambda$onOverflowIconClicked$1$OCFileListFragment (OCFileListFragment.java:417)
  at com.owncloud.android.ui.fragment.OCFileListFragment$$Lambda$1.onMenuItemClick (Unknown Source:4
  at android.widget.PopupMenu$1.onMenuItemSelected (PopupMenu.java:105)
  at com.android.internal.view.menu.MenuBuilder.dispatchMenuItemSelected (MenuBuilder.java:761)
  at com.android.internal.view.menu.MenuItemImpl.invoke (MenuItemImpl.java:172)
  at com.android.internal.view.menu.MenuBuilder.performItemAction (MenuBuilder.java:908)
  at com.android.internal.view.menu.MenuBuilder.performItemAction (MenuBuilder.java:898)
  at com.android.internal.view.menu.MenuPopup.onItemClick (MenuPopup.java:128)
  at android.widget.AdapterView.performItemClick (AdapterView.java:350)
  at android.widget.AbsListView.performItemClick (AbsListView.java:1683)
  at android.widget.AbsListView$PerformClick.run (AbsListView.java:4094)
  at android.widget.AbsListView$10.run (AbsListView.java:6582)
  at android.os.Handler.handleCallback (Handler.java:789)
  at android.os.Handler.dispatchMessage (Handler.java:98)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:6944)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:327)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1374)
```
and
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.deliverResults (ActivityThread.java:4491)
  at android.app.ActivityThread.handleSendResult (ActivityThread.java:4534)
  at android.app.ActivityThread.-wrap20 (Unknown Source)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1752)
  at android.os.Handler.dispatchMessage (Handler.java:105)
  at android.os.Looper.loop (Looper.java:164)
  at android.app.ActivityThread.main (ActivityThread.java:6940)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:327)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1374)
Caused by: java.lang.NullPointerException: 
  at com.owncloud.android.ui.fragment.OCFileListFragment.exitSelectionMode (OCFileListFragment.java:1590)
  at com.owncloud.android.ui.activity.FileDisplayActivity.exitSelectionMode (FileDisplayActivity.java:998)
  at com.owncloud.android.ui.activity.FileDisplayActivity.onActivityResult (FileDisplayActivity.java:977)
  at android.app.Activity.dispatchActivityResult (Activity.java:7556)
  at android.app.ActivityThread.deliverResults (ActivityThread.java:4487)
```

The code is as:
```
public void exitSelectionMode() {
        mActiveActionMode.finish();
}
```

So yes, we could just add a null-check while I still don't know why the variable would be null.

--- a/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java
+++ b/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java
@@ -1587,6 +1587,8 @@ public class OCFileListFragment extends ExtendedListFragment implements
      * Exits the multi file selection mode.
      */
     public void exitSelectionMode() {
-        mActiveActionMode.finish();
+        if (mActiveActionMode != null) {
+            mActiveActionMode.finish();
+        }
     }
 }

